Mathlib/Tactic/Nontriviality/Core.lean

```diff
@@ -55,7 +55,7 @@ def nontrivialityByAssumption (g : MVarId) : MetaM Unit := do
 
 /-- Attempts to generate a `Nontrivial α` hypothesis.
 
-The tactic first looks for an instance using `infer_instance`.
+The tactic first looks for an instance using `inferInstance`.
```

Comment:
I'm not sure if this is a correct change or not. Happy to revert.

Reply:
It could go either way, since `infer_instance` is the tactic and `inferInstance` is the term.

It's probably better to avoid the question entirely and improve the docstring:
```suggestion
The tactic first checks to see that there is not already a `Nontrivial α` instance
before trying to synthesize one using other techniques.
```